File: driver-service/src/main/java/com/dulfinne/taxi/driverservice/dto/response/DriverResponse.java
package com.dulfinne.taxi.driverservice.dto.response;

import lombok.Builder;

@Builder
public record DriverResponse(
    Long id,
    String username,
    String firstName,
    String lastName,
    String phoneNumber,
    String experience,
    CarResponse car,
    Double averageRating
) {
}

File: ride-service/src/main/kotlin/com/dulfinne/taxi/rideservice/client/dto/DriverResponse.kt
package com.dulfinne.taxi.rideservice.client.dto

class DriverResponse(
    val id: Long,
    val username: String,
    val firstName: String,
    val lastName: String,
    val phoneNumber: String,
    val experience: String,
    val car: CarResponse?,
    val averageRating: Double
)